bundles/admin/admin-announcements/publisher/AnnouncementsTool.js

```diff
@@ -35,13 +35,17 @@ Oskari.clazz.define('Oskari.admin.bundle.admin-announcements.publisher.Announcem
                 '</div>'),
             inputCheckbox: jQuery('<div><input type="checkbox" name="announcement"/><label></label></div>')
         };
+        this.noUI = true;
```

Comment:
Default to showing the UI instead?
```suggestion
        this.noUI = false;
```